source/CellMeasurer/CellMeasurer.js

```diff
@@ -1,35 +1,42 @@
 /** @flow */
-import { PureComponent } from 'react'
+import React from 'react'
 import { findDOMNode } from 'react-dom'
+import CellMeasurerCache from './CellMeasurerCache.js';
+
+type Children = (params: { measure: () => void }) => React.Children
```

Comment:
I don't think `React.Children` is what you think it is. (I think it must be explicitly imported via `import type {Children} from 'react'` in the current implementation of Flow.)